libraries/docker_installation_package.rb

```diff
@@ -128,9 +128,11 @@ def version_string(v)
                 end
 
       # https://github.com/seemethere/docker-ce-packaging/blob/9ba8e36e8588ea75209d813558c8065844c953a0/deb/gen-deb-ver#L16-L20
-      test_versioning = '3'
+      test_versioning = new_resource.version.to_f > 18.03 ? '3' : '1'
```

Comment:
change `new_resource.version.to_f` to `v.to_f`

Reply:
I completely missed that we where already passing this in 🤦‍♂️